repo_tools_data_schema/repo_tools_data_schema.py

```diff
@@ -34,7 +35,11 @@ def valid_email(s):
 @functools.lru_cache(maxsize=None)
 @backoff.on_exception(backoff.expo, SchemaError, max_time=60)
 def github_repo_exists(full_name):
-    resp = requests.get(f"https://api.github.com/repos/{full_name}")
+    headers = None
+    if (token := os.environ.get("GITHUB_TOKEN")):
+        headers = {"authorization": f"Bearer {token}"}
```

Comment:
This reminds me of a complication I encountered the other day: this header will be ignored if requests finds a .netrc with credentials in it.  It makes no sense, but here's the issue: https://github.com/psf/requests/issues/3929

The way to avoid this incorrect precedence is to use a session.

Reply:
I guess we don't need the defense, since anyone running this with a .netrc will have a token in .netrc that would also make this work.